### PR TITLE
fix(overlay): wait until after change detection to position overlays

### PR DESCRIPTION
**Pull request by jelbourn**

### CODE REVIEW

File: src/cdk/overlay/position/connected-position-strategy.ts [hunk 1]
```diff
@@ -147,6 +147,11 @@ export class ConnectedPositionStrategy implements PositionStrategy {
    * allows one to re-align the panel without changing the orientation of the panel.
    */
   recalculateLastPosition(): void {
+    // If the overlay had never been positioned before, do nothing.
```
Comment (kara) [inline src/cdk/overlay/position/connected-position-strategy.ts]: Nit: had -> has

File: src/cdk/overlay/overlay-ref.ts [hunk 2]
```diff
@@ -61,6 +62,13 @@ export class OverlayRef implements PortalHost {
       this._state.scrollStrategy.enable();
     }
 
+    // Update the position once the zone is stable so that the overlay will be fully rendered
+    // before attempting to position it, as the position may depend on the size of the rendered
+    // content.
+    first.call(this._ngZone.onStable.asObservable()).subscribe(() => {
+      this.updatePosition();
```
Comment (kara) [inline src/cdk/overlay/overlay-ref.ts]: The comment implies waiting for the zone to be stable before positioning, but it looks like we are also updating the position before the zone is stable on line 58. Is it intentional to update the position twice? If so, maybe update the comment to clarify.

Comment (crisbeto) [reply src/cdk/overlay/overlay-ref.ts]: The first call shouldn't be necessary. @jelbourn I ended up removing it.

Comment (jelbourn) [reply src/cdk/overlay/overlay-ref.ts]: Yeah, not sure why that was still there

**Comment by kara** [comment]
LGTM

**Comment by mmalerba** [comment]
please rebase

**Comment by crisbeto** [comment]
Rebased @mmalerba.

**Comment by mmalerba** [comment]
needs rebase again

**Comment by crisbeto** [comment]
Ready to go @mmalerba.

**Comment by mmalerba** [comment]
please rebase

**Comment by kara** [comment]
@crisbeto Can you check out this test failure? https://travis-ci.org/angular/material2/jobs/281616020

**Comment by crisbeto** [comment]
Should be ready to go now.

**Comment by jelbourn** [comment]
@kara can you describe the presubmit failure here?

**Comment by andrewseguin** [comment]
Needs CLA

**Comment by andrewseguin** [comment]
Also, not getting any presubmit errors. @kara do you know what they were?

**Comment by jelbourn** [comment]
@andrewseguin the CLA is fine, it's just me and Kristiyan

**Pull request merged**